nnpdf_data/nnpdf_data/commondata/ATLAS_WPWM_7TEV_46FB/filter.py

```diff
@@ -0,0 +1,31 @@
+"""
+When running `python filter.py` the relevant data yaml
+file will be created in the `nnpdf_data/commondata/ATLAS_WPWM_7TEV_46FB` directory.
+"""
+
+import yaml
+from filter_utils import get_data_values, get_kinematics
+
```

Comment:
Could you add 

yaml.add_representer(float, prettify_float)

so that the floats are rounded?

@Radonirinaunimi I think we should add this as part of the utils and import yaml from the utils so that it doesn't need to be added to each filter.

Reply:
Yes, I'll add that now